gateway-ha/src/main/java/io/trino/gateway/ha/resource/PublicResource.java

```diff
@@ -49,7 +49,7 @@ public Response getBackend(@PathParam("name") String name)
         try {
             ProxyBackendConfiguration backend = gatewayBackendManager
                     .getBackendByName(name)
-                    .get();
+                    .orElseThrow();
```

Comment:
I am not sure about this change.. kinda weird and implied but fine by me anyway.